source/release_notes/4.2.17-17.rst

```diff
@@ -0,0 +1,41 @@
+.. _PSMDB-4.2.17-17:
+
+================================================================================
+*Percona Server for MongoDB* 4.2.17-17
+================================================================================
+
+:Date: October 4, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.2/install/index.html>`_
+
+Percona Server for MongoDB 4.2.17-17 includes all features and bug fixes introduced in MongoDB 4.2.16 Community Edition and MongoDB 4.2.17 Community Edition.  
+It supports MongoDB 4.2.16 and 4.2.17 protocols and drivers.
+
+
+MongoDB Release Highlights
+=============================
+
+* The ``libsasl2`` dependencies are added for DEB-based operating systems to enable GSSAPI and Kerberos authentication to work out of the box after installing MongoDB
+* The number of modify updates in WiredTiger is now limited to prevent performance issues
+* Errors related to proxy connections receive new error codes which don't change the behavior of clients' drivers.
+
+Find the full list of changes in the  `MongoDB 4.2.16 Community Edition <https://docs.mongodb.com/manual/release-notes/4.2/#4.2.16---sep-13--2021>`_ and MongoDB 4.2.17 Community Edition <>`_ release notes.
+
+Percona Server for MongoDB Release Highlights
+=============================================
+
+* The ``mongod`` user is not deleted during the re-installation of Percona Server for MongoDB on RPM-based operating systems. This preserves the permissions to the MongoDB data directory for the ``mongod`` user as its user ID and group ID remain unchanged.
+* RPM packages now include the ``cyrus-sasl`` modules to enable LDAP authentication via native LDAP against Active Directory
```

Comment:
I suggest to keep only one Release Highlights section. Our goal is to deliver open source software to community and customers and if MongoDB corp team did most of the job - so be it :)

I feel that our improvement and a bug fix are not that significant for highlights. So in here it should be okay to keep only the ones which came from MongoDB corp, and our tickets should be okay to be under corresponding sections. 

I really like the value oriented description that you created in Release Highlights, we might reuse it in Improvements and Bugs Fixed sections. What do you think?

Reply:
Fixed: removed MongoDB highlights section, making it the only one. 
Extended PSMDB Jira issue subjects